pkg/build/nodeimage/build_impl.go

```diff
@@ -120,22 +120,6 @@ func (c *buildContext) buildImage(bits kube.Bits) error {
 		return err
 	}
 
-	// find the pause image and inject containerd config
```

Comment:
seems you don't need know to prePullImags in Line 117

Reply:
yeah, will fix after the conformance ga job is fixed